### PR TITLE
Ria 3220 Pay offline - Notifications when a PA appeal has been lodged and payment is pending

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-3220](https://tools.hmcts.net/jira/browse/RIA-3220)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationGeneratorConfiguration.java [hunk 1]
```diff
@@ -950,4 +950,23 @@
         );
     }
 
+    @Bean("submitAppealPayOfflineNotificationGenerator")
+    public List<NotificationGenerator> submitAppealPayOfflineLegalRepNotificationHandler(
```
Comment (bradsorour) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationGeneratorConfiguration.java:954]: Possibly best for us to rename this method to `submitAppealPayOfflineNotificationHandler` seeing we're sending out notifications now to both Legal Rep and Admin Officer.

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java [hunk 2]
```diff
@@ -837,4 +837,27 @@ private boolean isValidUserDirection(
             }, notificationGenerators
         );
     }
+
+    @Bean
+    public PreSubmitCallbackHandler<AsylumCase> submitAppealPayOfflineLegalRepNotificationHandler(
```
Comment (bradsorour) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java:842]: Same as above. Let's rename this method so it makes more sense.

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by Lee365 — DISMISSED** [review]

**Review by tarun-palisetty — DISMISSED** [review]

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**